cypress/integration/token-list/index.spec.ts

```diff
@@ -54,4 +54,10 @@ describe('Token list filter format control and theme control', function () {
       'rgb(244, 244, 246)'
     );
   });
+
+  it('has a responsive layout', () => {
+    cy.get('[data-cy="input-column"]').should('have.attr', 'min-width', '0px');
```

Comment:
I think the bug is that you're checking for `have.attr` but you should be checking for `have.css`